### PR TITLE
Update `Collator Set-up Guide` #8669r3e3n

### PR DESCRIPTION
**Pull request by obsessed-cake**

### CODE REVIEW

**Comment by KaiserKarel** [comment]
Task linked: [NX-109 Update `Collator Set-up Guide`](https://app.clickup.com/t/20465559/NX-109)

**Comment by obsessed-cake** [comment]
> Can this be updated to match https://docs.google.com/document/d/1clJnsYz1FoJ5Q5LP6-kadWWD6YrZ1lE_TtuUjbDeD9M/edit# ?

I requested access.

**Comment by Martin-Kiss** [comment]
@bostercf Please review to confirm #3286 or close #3263

File: docs/docs/developer-guides/collator-set-up-guide.md [hunk 1]
```diff
@@ -182,35 +186,59 @@ docker container run hello-world
...
-Save the following content to docker-compose.yml and run docker-compose up -d
+Save the following content to docker-compose.yml
 
 ```yml
 version: "3.7"
 
 services:
   composable_node:
-    image: composablefi/composable:latest
+    image: composablefi/composable:${COMPOSABLE_VERSION}
```
Comment (vimukthi-git) [inline docs/docs/developer-guides/collator-set-up-guide.md:196]: Why not use latest?

**Review by vimukthi-git — APPROVED** [review]
LGTM, though version should be the one that is marked as latest on composable repo
eg: https://github.com/ComposableFi/composable/releases/tag/release-v2.10009.1

**Pull request closed without merging**